src/account/PluginManagerInternals.sol

```diff
@@ -138,32 +133,21 @@ abstract contract PluginManagerInternals is IPluginManager {
         );
     }
 
-    function _installPlugin(address plugin, bytes32 manifestHash, bytes memory pluginInstallData) internal {
+    function _installPlugin(address plugin, PluginManifest calldata manifest, bytes memory pluginInstallData)
+        internal
+    {
         AccountStorage storage _storage = getAccountStorage();
 
         if (plugin == address(0)) {
             revert NullPlugin();
         }
 
-        // Check if the plugin exists.
-        if (_storage.pluginManifestHashes.contains(plugin)) {
-            revert PluginAlreadyInstalled(plugin);
```

Comment:
How is duplicated installation checked now?

Reply:
Do we need to check duplicate installation?

Reply:
Followup on this, I'm not sure if we actually really need to prevent duplicate installation, but from what I can tell the only place this would revert is if the plugin is installing an already installed execution function.

Otherwise, validation functions would just override storage (or add selectors) to the passed validation, and execution hooks would silently fail on set add.

Reply:
It's not checking on the plugin address level, it's checked within each component. Installs should still fail if they overlap at all.